thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/controllers/DashboardController.js

```diff
@@ -22,18 +22,22 @@ DashboardController.prototype = {
 
   handleAppEvent : function() {
     console.log("DashboardController.handleAppEvent");
-    mode = HASH_SERVICE.get(HASH_PARAMS.DASHBOARD_MODE);
+    const mode = HASH_SERVICE.get(HASH_PARAMS.DASHBOARD_MODE);
     console.log("tabName:" + tabName + " mode: " + mode);
-    var controllerName;
-    if (mode == constants.DASHBOARD_MODE_METRIC_SUMMARY) {
-      this.dashboardModel.tabSelected = "dashboard_metric-summary-tab";
-      controllerName = 'metricSummary';
-    } else if (mode == constants.DASHBOARD_MODE_ANOMALY_SUMMARY) {
-      this.dashboardModel.tabSelected = "dashboard_anomaly-summary-tab";
-      controllerName = 'anomalySummary';
-    } else if (mode == constants.DASHBOARD_MODE_WOW_SUMMARY) {
-      this.dashboardModel.tabSelected = "dashboard_wow-summary-tab";
-      controllerName = 'wowSummary';
+    let controllerName;
+    switch(mode){
+      case constants.DASHBOARD_MODE_METRIC_SUMMARY:
+        this.dashboardModel.tabSelected = 'dashboard_metric-summary-tab';
+        controllerName = 'metricSummary';
+        break;
+      case constants.DASHBOARD_MODE_WOW_SUMMARY:
+        this.dashboardModel.tabSelected = 'dashboard_wow-summary-tab';
+        controllerName = 'wowSummary';
+        break;
+      default:
+        this.dashboardModel.tabSelected = 'dashboard_anomaly-summary-tab';
+        controllerName = 'anomalySummary';
+        break;
```

Comment:
You don't need a `break` statement in a switch's `default` clause as there are no more cases to fall-through